_includes/pbc/all/install-features/202407.0/install-the-order-management-promotions-and-discounts-feature.md

```diff
@@ -0,0 +1,76 @@
+This document describes how to install the Order Management + Promotions & Discounts feature.
```

Comment:
Add this link to `Install Related features` - https://docs.spryker.com/docs/pbc/all/order-management-system/202407.0/base-shop/install-and-upgrade/install-features/install-the-order-management-feature.html#install-related-features